src/lib.rs

```diff
@@ -24,7 +24,7 @@ mod chain_sync;
 mod cli;
 mod cli_shared;
 mod daemon;
-mod db;
+pub mod db;
```

Comment:
`db` and `utils` should not be public. You most likely want to convert the integration test into a unit test.

Reply:
Moved.  Btw, since we are hosting forest CAR files now, what's the plan of providing libraries for manipulating forest CARs?

Reply:
There are third-party crates for reading and writing CAR files. I don't think we need to provide anything. The ForestCAR.zst format is internal, and I don't want other people to depend on it (for now).